app/views/content/international-teacher-training-to-english-standards-introducing-iqts.md

```diff
@@ -0,0 +1,177 @@
+
+## Six reasons to apply for iQTS
+
+1. iQTS leads to the award of English qualified teacher status ([QTS](https://www.gov.uk/guidance/qualified-teacher-status-qts)) with no further fees or training, and qualifies you to teach in all schools in England
+
+
+2. iQTS is open to international candidates from anywhere in the world who meet the eligibility criteria
+
+
+3. iQTS training and assessment are completed in your home country or school workplace, with no need to visit the UK
+
+
+4. iQTS offers high-quality, evidence-based training delivered by world class English teacher training providers, with in-school support from a trained mentor
+
+
+5. iQTS is equivalent to and meets the same teaching standards as QTS – regarded as the teacher training gold standard by British international schools and other schools and employers around the world
+
+
+6. Unlike an iPGCE, iQTS is a UK-government backed professional qualification which qualifies you to teach in all schools in England
+
+## Learn who iQTS is for
+
+iQTS is a teaching qualification designed for UK and non-UK citizens living outside the UK. If you are a UK or non-UK citizen living in England, you should [apply for teacher training in England](/steps-to-become-a-teacher). 
+
+iQTS is for you if:
+ 
+* you have no previous teaching qualifications or experience, and would like to gain a professional teaching qualification equivalent to QTS
+
+* you have some experience teaching, and would benefit from training and mentoring leading to a professional teaching qualification equivalent to QTS
+
+* you wish to teach in England, and you have a qualification (for example, a PGCE, PGDE or iPGCE) that does not allow you to teach in all schools in England
+
+* you are already a teacher, and you wish to develop professionally and enhance your global employability with an internationally recognised teaching qualification
+
+* you are already a teacher, and would benefit from a professional teaching qualification allowing you to teach in all schools in England
+
+## Apply for iQTS, step by step
+
+### 1. Check your eligibility
+
+To apply for iQTS you will need to have: 
+
+* a bachelor’s degree from a UK or non-UK university
+
+* a maths qualification equivalent to an English GCSE grade 4 
+
+* for teaching children aged 3 to 11, a biology, chemistry or physics qualification equivalent to an English GCSE grade 4 
+
+* excellent written and verbal English language skills
+
+#### Help with qualifications
+A GCSE (General Certificate of Secondary Education) is an academic qualification awarded for exams in England, usually taken at age 16. 
+
+If you’re not sure of the status of your degree in England, or whether your qualifications in maths, biology, chemistry or physics are equivalent to an English GCSE grade 4, visit [UK ENIC](https://www.ecctis.com/Default.aspx), the UK National Information Centre for global qualifications and skills.
```

Comment:
```suggestion
If you’re not sure of the status of your degree in England, or whether your qualifications are equivalent to an English GCSE grade 4, visit [UK ENIC](https://www.ecctis.com/Default.aspx). This is the UK National Information Centre for global qualifications and skills.
```
Removing the specific qualifications makes it a little more succinct and less wordy, feel the sentence still works without the extra info (have already said above specifically which subject qualifications they need)